solvation_analysis/rdf_parser.py

```diff
@@ -1,13 +1,18 @@
 import numpy as np
 import numpy.polynomial.polynomial
 import matplotlib.pyplot as plt
+from scipy.interpolate import interp1d, UnivariateSpline
 
 
-def quick_plot(bins, rdf):
-    plt.plot(bins, rdf, 'bo')
-    plt.show()
+def fit_data(bins, rdf):
+    f = UnivariateSpline(bins, rdf, k=4, s=0)
+    x = np.linspace(0, 10, num=100)
+    y = f(x)
+    return x, y
 
 
-def identify_minima(rdf, bins, n_minima):
-
-    return
+def identify_minima(bins, rdf):
+    f = UnivariateSpline(bins, rdf, k=4, s=0)
```

Comment:
This looks like a really promising approach. It is also readily expandable to any dimension with in a multidimensional spline basis.

The general case is any function that maps
```
f(rdf[:])-> critical_points[:]
```
I think splines are a really good candidate for this.

Reply:
The results are looking promising! My plan is to compose the function suggested above from the `interpolate_rdf` and `identify_minima` functions.

Reply:
perfecto!